pkg/platform/runtime/setup/setup.go

```diff
@@ -220,6 +235,35 @@ func (s *Setup) Update() (rerr error) {
 	return nil
 }
 
+func (s *Setup) getProcessesInUse() map[string]int32 {
```

Comment:
I think it makes more sense for this function to be part of the `osutils` package and we can supply it the directory to compare. Also we could have os-specific versions of these functions to reduce the if statements here.

Reply:
Okay, I moved it into the `osutils` package. However, when I tried splitting it up into OS-specific functions, I ended up with too much duplication. I left it as a single function.